### PR TITLE
Change .yaml files to new sources format

### PR DESCRIPTION
**Pull request by snowcrshd**
See https://github.com/open-power-host-os/builds/issues/111

### CODE REVIEW

**Comment by snowcrshd** [comment]
start tests

**Comment by snowcrshd** [comment]
start build

**Comment by alanoe** [comment]
We were talking about doing that just today, thanks!

**Comment by alanoe** [comment]
imho all theses patches could be merged into one to simplify git repository history, but that's minor and up to you

**Comment by snowcrshd** [comment]
@olavphibm updated the title.

I'll squash the commits after the review is done.

**Comment by alanoe** [comment]
I noticed gcc yaml file was the only yaml file which was not changed. That occurs because current source code retrieving capabilities are more restricted than the previous implementation due to security and we currently have not an equivalent for the commands used in the download_source attrib in gcc yaml file. I'd add a TODO in that yaml file mentioning it should be upgraded to new sources format as soon as we are able to clone gcc from git.

**Comment by olavphibm** [comment]
start build

**Comment by olavphibm** [comment]
start build

**Comment by snowcrshd** [comment]
start build

**Comment by olavphibm** [comment]
start build

File: golang/golang.yaml [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
- branch: 'release-branch.go1.7'
- expects_source: 'go'
- files:
+ sources:
+   - git:
+      src: 'https://github.com/golang/go.git'
+      commit_id: 'b67902fee7b56dd3235b2f489550c279248bacb9'
+      branch: 'release-branch.go1.7'
+      archive: 'go'
+ files
```
Comment (olavphibm) [inline golang/golang.yaml]: Missing `:`

**Comment by snowcrshd** [comment]
start build

**Comment by snowcrshd** [comment]
Will send a new PR removing `archive` tags when open-power-host-os/builds/pull/134 is merged.

**Pull request merged**